pkg/csi/manager.go

```diff
@@ -70,8 +77,36 @@ func (m *Manager) Run(cfg *config.Config) error {
 		return err
 	}
 
+	nodeID := cfg.NodeID
+
+	ifaces, err := sysfsnet.Interfaces()
+	if err != nil {
+		logrus.WithError(err).Warn("Failed to enumerate MAC addresses for VMI discovery")
```

Comment:
I saw you will return an error with discoverVMIName if we get an error here (which means `iface` is empty).

The error `errVMINotFound` should not present the actual error here, right?
Or could we return the error directly here?

Reply:
We could, but I thought by not returning an error here, the process could degrade gracefully by just reverting to its original behavior of attempting to use its original hostname.

Reply:
OK, that makes sense. Thanks!